### PR TITLE
convert state to hooks in color blindness file for a11y addon

### PR DESCRIPTION
**Pull request by Jessica-Koch**
Issue:

## What I did
Converted Colorblindness file in a11y addon to use the new react hooks api
## How to test

- Is this testable with Jest or Chromatic screenshots? yes, the tests should be the same
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: addons/a11y/src/components/ColorBlindness.tsx [hunk 1]
```diff
@@ -86,45 +79,39 @@ const getColorList = (active: string | null, set: (i: string | null) => void): L
   })),
 ];
 
-export class ColorBlindness extends Component<ColorBlindnessProps, ColorBlindnessState> {
-  state: ColorBlindnessState = {
-    active: null,
-  };
+export function ColorBlindness() {
```
Comment (shilman) [inline addons/a11y/src/components/ColorBlindness.tsx]: Is typing this redundant?

```
export const ColorBlindness: React.FC<{}>) = () => { ... }
```

Comment (Jessica-Koch) [reply addons/a11y/src/components/ColorBlindness.tsx]: I guess it depends if you want to explicitly type everything, I guess personal preference?

Comment (shilman) [reply addons/a11y/src/components/ColorBlindness.tsx]: Haha thanks! Wasn't requesting--was really a question about TypeScript best practices. I'm still a newbie.

Comment (Jessica-Koch) [reply addons/a11y/src/components/ColorBlindness.tsx]: @shilman sorry! it's still personal preference, Typescript should be able to figure it out, but it doesn't hurt.  If in the tsconfig, the `noImplicitReturns` rule is enforced, then you need to list the return values

**Comment by Jessica-Koch** [comment]
@shilman is there anything else i can do to get this to deploy?  I'm unable to view the !@now build failure, and i am unsure why it's failing on windows, i built this on windows...

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
@Jessica-Koch It's getting merged!!!

**Pull request merged**